misc/userscripts/qutebrowser_viewsource

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# 
+# This script fetches the unprocessed HTML source for a page and opens it in vim.
+#	:bind gf spawn --userscript qutebrowser_viewsource
+#
+
+path=/tmp/qutebrowser-src_$(date "+%s")
```

Comment:
Is there a reason you're not just using `$(mktemp)` here? Predictable tempfile names are a bad idea because of [symlink attacks](http://en.wikipedia.org/wiki/Symlink_race).